### PR TITLE
cmd.Env de-dup's differently as of Go 1.9

### PR DESCRIPTION
**Pull request by jhump**
I tried running `gorepoman fetch ...` after updating to Go 1.9, and it failed.

It fails because the `go get` command it uses internally has the wrong environment, due to a change in the `os/exec` package. This fixes it.

### CODE REVIEW

**Review by dragonsinth — APPROVED** [review]
```
Nobody intends to have duplicates anyway because it's so undefined
and everything handles it so poorly.
```

LGTM, but maybe we should just translate to a map and back.

File: util_test.go [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	"testing"
+)
+
+func TestCombineEnv(t *testing.T) {
+	orig := []string{
+		"A=1",
+		"B=2",
+		"C=3",
+		"D=",
+		"EFG=HIJKLMNOP",
```
Comment (dragonsinth) [inline util_test.go:14]: it'd be nice to have one test case where the value includes the `=` character

**Comment by dragonsinth** [comment]
nice

File: util_test.go (hunk 1 above)
Comment (jhump) [reply util_test.go:14]: https://github.com/fullstorydev/gorepoman/pull/9

**Pull request merged**